docker-compose.yml

```diff
@@ -693,6 +693,82 @@ services:
         aliases:
           - edx.devstack.xqueue_consumer
 
+  enterprise-catalog:
+    image: edxops/enterprise-catalog-dev
+    container_name: edx.devstack.enterprise.catalog
```

Comment:
Following the trend followed in the file (`edx.devstack.service-name`) it should be, others also needed to be updated
```
edx.devstack.enterprise-catalog
```